components/email.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * Manage BuddyPress Email Post Types.
+ *
+ * @since 1.6.0
+ */
+class BPCLI_Email extends BPCLI_Component {
+	/**
+	 * Create a new email post connected to an email type.
+	 *
+	 * ## OPTIONS
+	 *
+	 * --type=<type>
+	 * : Email type for the email (should be unique identifier, sanitized like a post slug).
+	 *
+	 * [--type-description=<type-description>]
```

Comment:
I don't see why the tax description is not required, considering almost all other fields are. Without it, the situation meta box and table columns will be empty.

Reply:
+1, let's make it required.